### PR TITLE
Changed docs

### PR DESCRIPTION
**Pull request by DenVilk**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Particularly user-facing changes**:

**PR Readiness Checklist**:

Complete these before marking the PR as `ready to review`:

- [ ] The PR title has a summary of the changes
- [ ] The PR body has a summary to reflect any significant (and particularly user-facing) changes introduced by this PR

### CODE REVIEW

File: docs/Configuration.md [hunk 1]
```diff
@@ -40,35 +40,60 @@ Usage of openvpn-auth-oauth2:
...
+CONFIG_HTTP_BASEURL=http://<vpn>:9000/
+CONFIG_HTTP_LISTEN=:9000
+# Define a random value with 16 or 24 characters
+CONFIG_HTTP_SECRET=1jd93h5b6s82lf03jh5b2hf9
+CONFIG_OPENVPN_ADDR=unix:///run/openvpn/server.sock
+CONFIG_OPENVPN_PASSWORD=<password from /etc/openvpn/password.txt>
+CONFIG_OAUTH2_ISSUER=https://company.zitadel.cloud
+CONFIG_OAUTH2_ENDPOINT_AUTH=https://company.zitadel.cloud/oauth/v2/authorize
+CONFIG_OAUTH2_ENDPOINT_TOKEN=https://company.zitadel.cloud/oauth/v2/token
+CONFIG_OAUTH2_ENDPOINT_DISCOVERY=https://company.zitadel.cloud/.well-known/openid-configuration
```
Comment (jkroepke) [inline docs/Configuration.md:93]: Does auto-discovery not work? Normally, define the CONFIG_OAUTH2_ISSUER should be good enough.

File: docs/Configuration.md [hunk 2]
```diff
@@ -40,35 +40,60 @@ Usage of openvpn-auth-oauth2:
...
+CONFIG_HTTP_TEMPLATE=/etc/sysconfig/auth.html
+CONFIG_OAUTH2_SCOPES=openid profile email offline_access
+CONFIG_LOG_LEVEL=debug
```
Comment (jkroepke) [inline docs/Configuration.md:96]: remove this line to keep it info level

File: docs/Configuration.md [hunk 3]
```diff
@@ -40,35 +40,60 @@ Usage of openvpn-auth-oauth2:
...
+CONFIG_OAUTH2_CLIENT_ID=34372461928374612@any
+CONFIG_OAUTH2_CLIENT_SECRET=ASDhjgadjhAUYSDGjkhasgdIATWDGJHASDtiwGDJAHSGDutwqdygASJKD12hfva
```
Comment (jkroepke) [inline docs/Configuration.md:98]: Atleast, please mask the secret here.

**Review by jkroepke — CHANGES_REQUESTED** [review]
Lood good in general. Found some nits.

Please also take an look on the DCO check, thanks!

File: docs/Configuration.md (hunk 3 above)
Comment (DenVilk) [reply docs/Configuration.md:98]: Its random string, not my code😅

File: docs/Configuration.md [hunk 4]
```diff
@@ -40,35 +40,56 @@ Usage of openvpn-auth-oauth2:
...
+```conf
+# Define the public http endpoint here.
+CONFIG_HTTP_BASEURL=http://<vpn>:9000/
+CONFIG_HTTP_LISTEN=:9000
+# Define a random value with 16 or 24 characters
+CONFIG_HTTP_SECRET=1jd93h5b6s82lf03jh5b2hf9
+CONFIG_OPENVPN_ADDR=unix:///run/openvpn/server.sock
+CONFIG_OPENVPN_PASSWORD=<password from /etc/openvpn/password.txt>
+CONFIG_OAUTH2_ISSUER=https://company.zitadel.cloud
+CONFIG_HTTP_TEMPLATE=/etc/sysconfig/auth.html
```
Comment (jkroepke) [inline docs/Configuration.md:91]:
```suggestion
```
This may not work for others.

File: docs/Providers.md [hunk 5]
```diff
@@ -57,3 +55,27 @@ After registering the app, you will receive an OAuth2 client ID and secret. Thes
...
+After created application, on page URLs you can find all links which you need.
+
+### Configuration
+- `CONFIG_HTTP_BASEURL=http://<vpn>:9000/`
+- `CONFIG_HTTP_LISTEN=:9000`
+- `CONFIG_HTTP_SECRET=1jd93h5b6s82lf03jh5b2hf9`
+- `CONFIG_OPENVPN_ADDR=unix:///run/openvpn/server.sock`
+- `CONFIG_OPENVPN_PASSWORD=<password from /etc/openvpn/password.txt>`
+- `CONFIG_OAUTH2_ISSUER=https://company.zitadel.cloud`
+- `CONFIG_HTTP_TEMPLATE=/etc/sysconfig/auth.html`
```
Comment (jkroepke) [inline docs/Providers.md:78]:
```suggestion
```

This may not work for others.

**Review by jkroepke — APPROVED** [review]
Thanks!

**Pull request merged**